migration.md

```diff
@@ -0,0 +1,89 @@
+# Migration guide
+
+This page documents key considerations when migrating an existing application using legacy Hyperledger Fabric client SDKs to the Fabric Gateway client API.
+
+## Fabric programming model
+
+The Fabric Gateway client API is an evolution of the legacy SDKs and of the Fabric programming model. The API concepts, structure and behavior remain broadly the same as the legacy SDKs.
+
```

Comment:
add "The following terminology applies to this migration guide:"

Reply:
This is supposed to be listing common elements of both the legacy and new APIs, so it's clear what **isn't** different. The intention isn't to define terminology. Users of legacy SDKs should already be familiar with this common terminology

Reply:
I've added a _"Similarities include:"_ statement to begin the list to (hopefully) clarify the intent